base/applications/rapps/include/unattended.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
 #define CMD_KEY_APPWIZ L"APPWIZ"
+#define CMD_KEY_GENINST L"GENINST"
+#define CMD_KEY_UNINSTALL L"UNINSTALL"
```

Comment:
Those two don't have an entry in the UsageString below?
(Also it may be logical to put the uninstall after the install, not before.)

Reply:
GENINST is for internal Rapps usage, after downloading the .zip, it starts a new child process that acts as the installer.

UNINSTALL is for the uninstalled command written to the registry. However, this command is generic enough that it can uninstall other apps and could be documented for people to use.

Reply:
I added a usage string for /UNINSTALL